bundles/org.openhab.binding.coronastats/src/main/java/org/openhab/binding/coronastats/internal/dto/CoronaStatsCases.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.coronastats.internal.dto;
+
+import static org.openhab.binding.coronastats.internal.CoronaStatsBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+
+/**
+ * The {@link CoronaStatsCountry} class holds the internal data representation of each Country
+ *
+ * @author Johannes Ott - Initial contribution
+ */
+@NonNullByDefault
+public class CoronaStatsCases {
+    private int cases = -1;
+
```

Comment:
Please remove the empty lines between the fields.

Reply:
Removed